src/openms/source/ANALYSIS/XLMS/OPXLHelper.cpp

```diff
@@ -1070,7 +1052,64 @@ namespace OpenMS
         {
           ph_alpha.setMetaValue("XL_Protein_position_beta", "-");
         }
+      }
+    }
+  }
+
+  void OPXLHelper::addBetaAccessions(std::vector< PeptideIdentification > & peptide_ids)
+  {
+    for (PeptideIdentification& id : peptide_ids)
+    {
+      PeptideHit& ph_alpha = id.getHits()[0];
```

Comment:
what happens if  its empty?

```suggestion
      if (id.getHits().empty()) continue;
      PeptideHit& ph_alpha = id.getHits()[0];
```